src/util/records.py

```diff
@@ -514,15 +514,15 @@ def remove_group(cr, xml_id=None, group_id=None):
         if on_delete_action == "r":
             if foreign_table not in standard_tables:
                 cr.execute(
-                    'SELECT COUNT(*) FROM "{}" WHERE "{}" = %s'.format(foreign_table, foreign_column),
+                    "SELECT COUNT(*) FROM {} WHERE {} = %s".format(foreign_table, foreign_column),
                     (group_id,),
                 )
                 count = cr.fetchone()[0]
                 if count:
                     custom_tables.append((foreign_table, foreign_column, count))
                 continue
 
-            query = 'DELETE FROM "{}" WHERE "{}" = %s'.format(foreign_table, foreign_column)
+            query = "DELETE FROM {} WHERE {} = %s".format(foreign_table, foreign_column)
             query = cr.mogrify(query, (group_id,)).decode()
 
             if column_exists(cr, foreign_table, "id"):
```

Comment:
Will be always false for quoted table names.
We have multiple option here:
 - ignore it and don't explode the query in this case.
 - unquote the table name
 - call `get_fk` with `quote_ident=False`

Reply:
What about using identifiers from `psycopg2.sql`?

Reply:
It's a more intrusive change that needs more tests that this simple patch.
But feel free to start experimenting.